modules/dynamicDate.js

```diff
@@ -0,0 +1,7 @@
+import { DateTime } from './luxon.min.js';
+
+export default function updateDateTime() {
+  const dateTimeElement = document.getElementById('date-para');
+  const newTime = DateTime.now();
+  dateTimeElement.textContent = newTime.toFormat('dd-MM-yyyy HH:mm:ss');
```

Comment:
- [ ] 👍 Well! Functionally, you've successfully added the current date, and visually, the date is visible. However, ❌ your date and time format doesn't follow the wireframe. 😕 Please consider following the wireframe and updating your code to match the design as much as possible. 🔄

| Your | Design |
| --- | --- |
| ![image](https://github.com/Armynerh/awesome_books-es6/assets/55840999/4284e48d-4c35-4751-b562-c12fe8fe5605) | ![image](https://github.com/Armynerh/awesome_books-es6/assets/55840999/b5593394-5be2-4ea2-973a-416b7a1df8ee) |